Basic/APIDiorama/Assets/Tests/Editor/ProjectTests.cs

```diff
@@ -0,0 +1,19 @@
+using NUnit.Framework;
+
+namespace Unity.Netcode.Samples.APIDiorama.Tests.Editor
+{
+    public class ProjectTests
```

Comment:
Actually I thought it'd be picked up automatically, I'll make another commit to address this. Thanks for flagging!

Reply:
IIRC project tests are only run on branches that target develop and main. I believe it actually is working. We could configure our branch rules and enable this branch to run tests.

Reply:
I did that just now and it's not kicking off the actual tests, and I believe that's because we'd have to enable that inside of the build.yml job def file, since that's where we've explicitly specified for tests to run on PRs targeting develop and main. This job will not do anything for this PR since it doesn't target any of those.
So we could modify the build.yml file to allow tests for the `sample/api-diorama` branch as well? (though this shouldn't be pushed though).
@SamuelBellomo

Reply:
ah right I missed the branch. since the base api-diorama branch will itself merge on develop, I'm ok keeping our setup the way it is. CI checks are there to make sure we don't bork our develop branch and block everyone working off develop. Since this feature branch will be mostly a solo dev on it, having CI validation after merge to base api-diorama branch should be fine, it'll just be a sanity check for Paolo.

Reply:
Okay so correction on what I mentioned here:

> I believe that's because we'd have to enable that inside of the build.yml job def file, since that's where we've explicitly specified for tests to run on PRs targeting develop and main. This job will not do anything for this PR since it doesn't target any of those.

Adding the branch rule (which we can remove after this PR) for `sample/api-diorama` branch actually _did_ trigger tests, as can be seen with the other [PR](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/pull/116) targeting `sample/api-diorama`. I think it's a bug that Yamato didn't kick off tests for this PR, in my opinion.

However, the editor tests that have been created in this PR won't be triggered automatically by Yamato, as seen [here](https://unity-ci.cds.internal.unity3d.com/job/22830390/dependency-graph/overview). The actual definition of this job is inside the build.yml file. It simply builds every bitesize sample for both win and mac.

The distinction inside of the Boss Room job def file is that we run [project tests](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/blob/main/.yamato/_triggers.yml) instead, which pack and run tests for each platform. 

You can see the Yamato flow of dependencies for an open Boss Room PR [here](https://unity-ci.cds.internal.unity3d.com/job/18743866/dependency-graph), as an example.

So here we'd have to modify our Yamato job def files to test our bitesize projects for their respective Runtime & Editor tests.

Reply:
(And to build on above, I manually kicked off the Pull Request Trigger job manually through Yamato for this PR, since I'm pretty sure Yamato never did)

Reply:
All right, thanks for investigating! So can I merge this branch in API Diorama for now?

Reply:
Integrating CI would be a blocker for merging to develop, not to the api-diorama branch. It's up to you how you want to split this, I don't mind (and already approved this PR)

Reply:
I second that. So long as another PR modifies the actual job def files, this one can go in for now.